### PR TITLE
Add closing of logger file handlers to tearDown method in test_input.py

### PR DESCRIPTION
**Pull request by mulhod**

### CODE REVIEW

**Comment by mulhod** [comment]
Let's see if this works.

**Review by desilinguist — APPROVED** [review]
LGTM!  👍

**Comment by desilinguist** [comment]
I think we can merge this without a second review given the very limited scope of the bugfix and the fact that the broken build now passes.

**Pull request merged**
